blocks/leadspace/leadspace.css

```diff
@@ -1,6 +1,10 @@
 /* Leadspace block styles */
 
 /** Section styling */
+.section.leadspace-container + .section.columns-container {
```

Comment:
Isn't a `Leadspace` always meants to be in its own section? Or is this handling a different use case?

Reply:
I don't understand the question. Leadspace is its own block, so yes it is in its own section. The person who created the block appended ".section" in front of his code, so I left that, if that's your question?

Reply:
OH -- I think I understand. The problem I am solving with this is the fact that the Figma layouts have a non-standard spacing between leadspace and the first columns container when they are together. Height is only 70 in those cases. 
![image](https://user-images.githubusercontent.com/3883795/228013782-e0f4c4d4-5392-4d04-a90e-8903e0831601.png)

Reply:
The Figma designs say that 70px should be 96px. 
![image](https://user-images.githubusercontent.com/3883795/228015141-ae0032c5-dec0-4765-8e45-1fdc1e267853.png)

Reply:
I wonder where adding this style makes the most sense. Is it a Leadspace style or is it a columns style? Or is it a common style when leadspace and column sections are next to each other?

Reply:
So far it is only when lead space is immediately followed by a Columns block, then the padding between them won't be the 96px as show in the Figma grid layout, instead it will match their actual Figma page mockups.

This may end up becoming the default no matter what follows the Leadspace block, but I wasn't knowledgeable enough about the content to know right now. Feel free to change later on :-)